LandSandBoat/server
# Module to Remove Guild Kits

**_I affirm:_**

## What does this pull request do?

Could do other things with this module as well but this is basic just remove the crafting kits from the guild stock.

## Steps to test these changes

Load module and see that there are no more kits on the guild shops 🥳 
Before
![img_20220615_232300](https://user-images.githubusercontent.com/6879206/173984358-eea9e710-dc0b-48ba-b4ce-c4bfa05a72e5.jpg)
After
![img_20220615_232306](https://user-images.githubusercontent.com/6879206/173984303-fc65b758-a9ab-4cb4-b026-4144da453cbd.jpg)
